### PR TITLE
rework level field to be better(i think)

### PR DESCRIPTION
**Pull request by ezrn**

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java [hunk 1]
```diff
@@ -35,14 +36,14 @@ public String getName() {
     @Override
     public String[] getDescription(int level) {
         return new String[]{
-                "For every enemy within <stat>" + radius + "</stat> blocks,",
-                "You deal <val>" +  (10 + ((level - 1) * 5)) + ".0%</val> extra damage",
+                "Within <stat>" + radius + "</stat> blocks",
+                "For every enemy that outnumbers you,",
+                "you take and deal <val>" + (damagePerPlayer * level) + "</val> extra damage",
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java:41]: You take more damage when you are outnumbered?

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java [hunk 2]
```diff
@@ -35,14 +36,14 @@ public String getName() {
...
                 "",
-                "For every ally <stat>" + radius + "</stat> blocks,",
-                "You deal <val>" + (10 + ((level - 1) * 5)) + ".0%</val> less damage",
+                "For every enemy that you outnumber,",
+                "you take and deal <val>" + (damagePerPlayer * level) + "</val> less damage",
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java:44]: You take less damage when you outnumber them?

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java [hunk 3]
```diff
@@ -35,14 +36,14 @@ public String getName() {
...
-                "For every enemy within <stat>" + radius + "</stat> blocks,",
-                "You deal <val>" +  (10 + ((level - 1) * 5)) + ".0%</val> extra damage",
+                "Within <stat>" + radius + "</stat> blocks",
+                "For every enemy that outnumbers you,",
+                "you reduce and deal <val>" + (damagePerPlayer * level) + "</val> extra damage",
                 "",
-                "For every ally <stat>" + radius + "</stat> blocks,",
-                "You deal <val>" + (10 + ((level - 1) * 5)) + ".0%</val> less damage",
+                "For every enemy that you outnumber,",
+                "you reduce and deal <val>" + (damagePerPlayer * level) + "</val> less damage",
```
[the remark this answers is not in the document]
Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java:44]: reduce less = bad, deal less = bad

reduce more = good, deal more = good

I could change it to be deal and take, but then I would have to make 4 of the same exact number appear which is kinda dumb. Maybe I could rewrite it another way too.

[the remark this answers is not in the document]
Comment (braulio-dev) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java:44]: Why is the ability punishing for having extra allies though. I'd understand it if it negated the effects of the ability but making you deal less damage makes it seem like an ability for people who are playing by themselves

[the remark this answers is not in the document]
Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java:44]: This is how the extra damage is calculated:
double damageAdded = Math.max(0, (nearbyDifference * (damagePerPlayer * level)));
Meaning that the extra damage cannot be negative and the player is never punished for being around their allies.
The benefit of the skill comes into play while you are outnumbered, but it never puts you at a disadvantage when with your allies

[the remark this answers is not in the document]
Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java:44]: "You cannot go below your weapons base damage",

[the remark this answers is not in the document]
Comment (braulio-dev) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java:44]:
```
For every outnumbering nearby enemy,
you deal X more and take X less damage.
```

You could just say that it's implied that you only get the effects for the other team having X more players, the damage can't go below the base, and having more allies counteracts the effects.

[the remark this answers is not in the document]
Comment (Mykindos) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java]: image
now its just better version of intimidation 😠

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java [hunk 4]
```diff
@@ -35,14 +36,8 @@ public String getName() {
...
-                "For every enemy within <stat>" + radius + "</stat> blocks,",
-                "You deal <val>" +  (10 + ((level - 1) * 5)) + ".0%</val> extra damage",
-                "",
-                "For every ally <stat>" + radius + "</stat> blocks,",
-                "You deal <val>" + (10 + ((level - 1) * 5)) + ".0%</val> less damage",
-                "",
-                "Maximum extra damage: <stat>60%",
-                "Minimum extra damage: <val>"+ (60 - ((level - 1) * 15)),
+                "For every enemy that you are outnumbered by within <stat>" + radius + "</stat> blocks,",
+                "you deal <val>" + (level * damagePerPlayer) + "</val> more and take <val>" + (level * damagePerPlayer) + "</val> less damage",
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java:40]: Bro doesnt english

For every enemy that you are outnumbered by within x blocks, you:

Deal y more damage
Take z less damage

-- Any other information that might be relevant

**Pull request closed without merging**